src/simulator/RangeSensor.tsx

```diff
@@ -54,12 +54,17 @@ const RangeSensor = ({
     },
     [isFocused]
   );
+  const valuePercent = useMemo(
+    () => ((value - min) / (max - min)) * 100,
+    [min, max, value]
+  );
```

Comment:
I suspect this calculation is very low cost compared with the bookkeeping around useMemo. Can it just be a variable calculated each render?